app/js/form.js

```diff
@@ -0,0 +1,183 @@
+
+$('input,textarea').on('input',function(){
+  var inpObj1 = document.getElementById("name").checkValidity();
+  var inpObj2 = document.getElementById("email").checkValidity();
+  var inpObj3 = document.getElementById("age").checkValidity();
+  var inpObj4 = document.getElementById("dropdown").checkValidity();
+  var inpObj5 = document.getElementById("most-like").checkValidity();
+  var inpObj7 = document.getElementById("comment").checkValidity();
+  console.log(inpObj7);
+  var requiredCheckboxes = $('input[type="checkbox"]');
+  //console.log('6');
+  //requiredCheckboxes.change(function(){
+        //console.log("message")
+  if(requiredCheckboxes.is(':checked')) {
+            requiredCheckboxes.removeAttr('required');
+            //requiredCheckboxes.removeAttr('oninvalid');
+            //requiredCheckboxes.removeAttr('oninput');
+            inpObj6=true
+            //onsole.log('4');
+        }
+
+ else {
+            requiredCheckboxes.attr('required', 'required');
+            //requiredCheckboxes.attr('oninvalid','Invalid entry');
+            //requiredCheckboxes.attr('oninput','');
+            //console.log('5');
+            inpObj6=false
+        }
+
+    console.log(inpObj1,inpObj2,inpObj3,inpObj4,inpObj5,inpObj6,inpObj7);
+    if(inpObj1 && inpObj2 && inpObj3 && inpObj4 && inpObj5 && inpObj6 && inpObj7)
+    {
+      $('#submit').css('background','#37af65')
+      $('#submit').css('color','#f3f3f3')
+      console.log('76');
```

Comment:
Make a unique class and add dynamically using jquey.

Reply:
Done